<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPlainTextEdit" name="textPreView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>1001</width>
     <height>371</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnGenerateScript">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>生成脚本</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Qt目录：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Qt版本：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Qt编译器：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>MingW:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="letQtPath">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>C:\Qt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxQtVersion"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbxQtBuilder"/>
      </item>
      <item>
       <widget class="QLineEdit" name="letMingW">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>C:\Qt\Tools\mingw1120_64</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pbtnQtPath">
        <property name="text">
         <string>选择Qt目录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbtnMingW">
        <property name="text">
         <string>MingW目录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>601</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Mysql目录：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Qt Cmake：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Qt Ninja：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Visual Studio:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="letMysqlPath">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>C:\Program Files\MySQL\MySQL Server 8.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="letQtCmake">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>C:\Qt\Tools\CMake_64\bin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="letNinja">
        <property name="text">
         <string>C:\Qt\Tools\Ninja</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="letVisualStudio">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>C:\Program Files\Microsoft Visual Studio\2022\Community</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pbtnMysqlPath">
        <property name="text">
         <string>选择Mysql目录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbtnVisualStudio">
        <property name="text">
         <string>Visual Studio目录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lableNotify">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnSaveText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存脚本</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
